doc/api/repl.md

```diff
@@ -151,34 +151,18 @@ global or scoped variable, the input `fs` will be evaluated on-demand as
 
 <!-- YAML
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/55204
+    description: The `process.setUncaughtExceptionCaptureCallback()`
```

Comment:
This is an implementation detail. Does it change the behavior of the function? If not, we should remove it from here. If it changes the behavior, this description should mention the behavior change rather than implementation specifics.